.github/workflows/build-continuous-linux.yml

```diff
@@ -10,19 +9,20 @@ jobs:
       - uses: actions/checkout@v2.3.4
       - name: Install Qt5
         run: |
-          sudo apt-get install -y build-essential gettext qttools5-dev libqt5svg5-dev libqt5xmlpatterns5-dev
+          # register kde neon repository
+          echo 'deb http://archive.neon.kde.org/user focal main' | sudo tee /etc/apt/sources.list.d/neon.list
+          curl https://origin.archive.neon.kde.org/public.key -o - | sudo apt-key add -
+          
+          # install build dependencies
+          sudo apt-get update
+          sudo apt-get install -y build-essential gettext git qt5-default qttools5-dev libqt5svg5-dev libqt5xmlpatterns5-dev
       - name: build
         run: |
           sudo wget -q https://github.com/AppImageCrafters/appimage-builder/releases/download/v0.8.8/appimage-builder-0.8.8-4e7c15f-x86_64.AppImage -O /usr/local/bin/appimage-builder
           sudo chmod +x /usr/local/bin/appimage-builder
           appimage-builder --recipe dist/appimage-builder-recipe.yml --skip-test
-
-      - name: upload Seamly2D.AppImage
-        uses: actions/upload-release-asset@v1.0.2
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - uses: actions/upload-artifact@v2
```

Comment:
ok so this is a trial version that runs in your repo?

Reply:
Yes, those binaries are being hosted in my fork.

Once merged this should create an AppImage on every push done to the Seamly2D repository. This will allow you to do some testing before releasing (or merging) it.